tests/dataframe/unit/test_functions.py

```diff
@@ -1278,7 +1278,7 @@ def test_concat(self):
         col = SF.concat(SF.col("cola"), SF.col("colb"))
         self.assertEqual("CONCAT(cola, colb)", col.sql())
         col_single = SF.concat("cola")
-        self.assertEqual("cola", col_single.sql())
+        self.assertEqual("CONCAT(cola)", col_single.sql())
```

Comment:
This isn't reduced to `cola` because `SF` functions instantiate the expressions manually instead of parsing them, so since the generator now doesn't override `concat_sql` to do the `len(..) == 1` check, it outputs it as is. Not sure if this is ok or not.